system/I18n/TimeTrait.php

```diff
@@ -699,9 +699,7 @@ public function setTimezone($timezone)
     #[ReturnTypeWillChange]
     public function setTimestamp($timestamp)
     {
-        $time = date('Y-m-d H:i:s', $timestamp);
-
-        return self::parse($time, $this->timezone, $this->locale);
+        return parent::setTimestamp($timestamp);
```

Comment:
the whole method `setTimestamp` can just be removed since it only return same parent method?

Reply:
Removed.